### DISCUSSION TITLE
how to set netlist 3s2p

### DISCUSSION THREAD

**Discussion by chuckliu1979** [pybamm-team/liionpack#299] [Q&A]
I want to analysis 3s2p netlist， i see liionpack only create netlist with mpns, parallels first then series. how to series first then parallel ?

**Comment by TomTranter** [comment]
Hi, yes you can do both. Maybe the syntax should be updated to accept the string as 3s2p or 2p3s like you suggest. At the moment here is an example https://github.com/pybamm-team/liionpack/blob/develop/examples/different_confiurations.py
